backend/src/xfd_django/xfd_api/views.py

```diff
@@ -1335,12 +1339,39 @@ async def call_search_vulnerabilities(
         return VulnerabilitySearchResponse(result=vulnerabilities, count=count)
 
     try:
-        # Convert each ORM instance to a Pydantic model
+        for v in vulnerabilities:
```

Comment:
Move this to the search_vulnerabilities function in api_methods to keep views.py manageable to read.